cylc/flow/graphnode.py

```diff
@@ -89,7 +90,13 @@ def parse(self, node):
 
         Return:
             tuple:
-            (name, offset_is_from_icp, offset_is_irregular, offset, output)
+            (name, offset, output,
+            offset_is_from_icp, offset_is_irregular, offset_is_absolute)
+
+        TODO: offset_is_from_icp is '^' or None - should be boolean?
```

Comment:
@hjoliver - does this need addressing still?

Reply:
Good spotting.  The variable name suggests we thought the value is boolean.  It's actually of no consequence as (I checked) the variable is only used in `if` tests (where '^' vs `None` behaves just like `True` vs `False).  However, I'll follow-up with a small change to make the code less obtuse.